test/testStatuspage.js

```diff
@@ -16,6 +16,8 @@ const assert = require('assert');
 const shell = require('shelljs');
 const sinon = require('sinon');
 
+process.env.HELIX_FETCH_FORCE_HTTP1 = 'true';
```

Comment:
Why not a real boolean value ? With the current code, the `if` tests the string... 

```suggestion
process.env.HELIX_FETCH_FORCE_HTTP1 = true;
```

Reply:
@kptdobe copied from the master 😉 : https://github.com/adobe/openwhisk-action-builder/blob/b36b2a61c60a222b967b77decfcd5d637b8c4896/test/deploy.test.js#L23

I guess the value doesn't really matter, the env var just needs to be present, i.e. not `undefined`...

Reply:
technically, it's not possible to set a boolean value via the environment. so I'd rather mimic what the process would receive from the environment.